public/electron.js

```diff
@@ -37,6 +37,13 @@ const btcdSettingsDir = path.join(isDev ? 'data' : userDataPath, 'btcd');
 const lndArgs = process.argv.filter(a =>
   /(^--bitcoin)|(^--btcd)|(^--neutrino)/.test(a)
 );
+let lndIP = 'localhost';
+process.argv.filter(a => {
+  if (/(^--lndip)/.test(a)) {
+    let split = a.split('=');
+    if (split.length > 1) lndIP = split[1];
+  }
+});
```

Comment:
Move cli arg parsing to a function in `helpers.js`. This should be something like:
```
let lndIP = parseCliArg('lndip');
```

Reply:
fixed